src/test/groovy/org/gradle/profiler/report/JsonResultWriterTest.groovy

```diff
@@ -41,152 +41,155 @@ class JsonResultWriterTest extends Specification {
         when:
         writer.write([result1, result2], stringWriter)
         then:
-        stringWriter.toString() == """{
-  "scenarios": [
-    {
-      "definition": {
+        stringWriter.toString() ==~ /\{
+  "environment": \{
+    "profilerVersion": ".*"
```

Comment:
The version should be available here, so you could assert the actual version string, right?

Reply:
It is `UNKNOWN` if you try to run the test from classes. But... yeah, we could do it that way.